DESCRIPTION

```diff
@@ -0,0 +1,32 @@
+Package: popgenInfo
+Title: popgenInfo is not really an R package, but this is here to list the dependencies for running all the vignettes.
+Version: 0.1
+Authors@R: person("First", "Last", email = "first.last@example.com",
```

Comment:
This was intentional, but I would like your thoughts as to whether or not you want us to put our names in there.

It's really only a formality as R packages need to have an Authors field in the description.

> On Jan 27, 2016, at 09:46 , Hilmar Lapp <notifications@github.com> wrote:
> 
> In DESCRIPTION <https://github.com/NESCent/popgenInfo/pull/123#discussion_r51020556>:
> 
> > @@ -0,0 +1,32 @@
> > +Package: popgenInfo
> > +Title: popgenInfo is not really an R package, but this is here to list the dependencies for running all the vignettes.
> > +Version: 0.1
> > +Authors@R: person("First", "Last", email = "first.last@example.com",
> Did you leave this intentionally "blank" (so as to be filled in later by everyone, for example?), or was this an oversight?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/NESCent/popgenInfo/pull/123/files#r51020556>.
>